packages/fxa-admin-server/src/graphql.ts

```diff
@@ -34,6 +34,7 @@ export enum BounceSubType {
 
 export interface EmailBounce {
     email: string;
+    template: string;
```

Comment:
I'm not really sure how to do this with gql, but we have an `emailTypes` table that has all our templates already. We might be able to use that instead of storing a string.